nodejs/dist/package.json

```diff
@@ -2,10 +2,14 @@
   "name": "uws",
   "version": "0.3.1",
   "scripts": {
-    "build": ""
+    "preinstall": "node-gyp rebuild && exit 0; echo 'Falling back to precompiled binaries';"
```

Comment:
Is this really going to work? If I type node-gyp in my terminal it asks me if I should install it and it expects y/N keyboard input to continue.